pkg/render/logstorage/esgateway/esgateway.go

```diff
@@ -77,6 +80,9 @@ type Config struct {
 	TrustedBundle              certificatemanagement.TrustedBundle
 	ClusterDomain              string
 	EsAdminUserName            string
+
+	// Whether or not the cluster supports pod security policies.
```

Comment:
// Whether or not
is not a correct phrase in this context, as or not is redundant.

// Whether

Reply:
Comments are updated.